src/api/index.ts

```diff
@@ -1,33 +1,32 @@
-import express, { NextFunction, Request, Response } from "express";
-import { check, oneOf, validationResult } from "express-validator";
+import express, {NextFunction, Request, Response} from "express";
+import {check, oneOf, validationResult} from "express-validator";
```

Comment:
Can you make sure your IDE has the editorconfig and eslint plugin installed? All these whitespace changes shouldn't happen.